web/packages/teleport/src/Bots/List/ActionCell.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { Cell } from 'design/DataTable';
+import { MenuButton, MenuItem } from 'shared/components/MenuAction';
+
+import { BotOptionsCellProps } from 'teleport/Bots/types';
+
+export function BotOptionsCell({
+  bot,
+  onEdit,
+  onDelete,
+}: BotOptionsCellProps): React.JSX.Element {
+  return (
+    <Cell align="right">
+      <MenuButton>
+        <MenuItem disabled onClick={() => onEdit(bot)}>
+          Edit...
```

Comment:
Can we remove the `...`? cc @roraback

Reply:
Either way, I'll handle the PR with the new designs. I'll add a comment to the Figma: https://www.figma.com/file/v6GunK50D2VC7w7I2FBDNf?node-id=3033:32677&mode=dev#674802580